distributions/demo-server/.gitignore

```diff
@@ -0,0 +1 @@
+env.secrets
```

Comment:
Should we add **/env.secrets in case folks put it somewhere deeper in the directory?

Reply:
Sure done.